git-hooks/install.bash

```diff
@@ -1,10 +1,11 @@
 #! /usr/bin/env bash
 
 # install the global hooks
-git config --global core.hooksPath "$(dirname "${BASH_SOURCE[0]}")"/git_hooks/
-
-if dpkg --compare-versions "$(git --version | awk '{print $3}')" lt 2.9
-then
-    tue-install-info "Running sudo apt-get install --assume-yes git to upgrade git>2.9"
+if dpkg --compare-versions $(git --version | awk '{print $3}') lt 2.9; then
+    echo "Need to upgrade git to use the git-hooks feature"
+    tue-install-ppa ppa:git-core/ppa
+    echo "sudo apt-get install git"
     sudo apt-get install --assume-yes git
```

Comment:
`tue-install-ppa`, adds to a list. It is added in the end. So running apt-get install here will not work.

Reply:
Good point...